src/pages/home/HeaderView.tsx

```diff
@@ -288,7 +290,9 @@ function HeaderView({report, parentReportAction, reportID, onNavigationMenuButto
                             </PressableWithoutFeedback>
                             <View style={[styles.reportOptions, styles.flexRow, styles.alignItemsCenter]}>
                                 {!shouldUseNarrowLayout && isChatUsedForOnboarding && freeTrialButton}
-                                {isTaskReport && !shouldUseNarrowLayout && ReportUtils.isOpenTaskReport(report, parentReportAction) && <TaskHeaderActionButton report={report} />}
+                                {isTaskReport && !shouldUseNarrowLayout && ReportUtils.isOpenTaskReport(report, parentReportAction) && Task.canActionTask(report, accountID) && (
+                                    <TaskHeaderActionButton report={report} />
+                                )}
```

Comment:
```suggestion
                               {isTaskReport && !shouldUseNarrowLayout && ReportUtils.isOpenTaskReport(report, parentReportAction) && <TaskHeaderActionButton report={report} />}
```
We should revert the condition checking `canActionTask` to display the `TaskHeaderActionButton`, as this component already includes logic to handle disabling based on [canActionTask](https://github.com/Krishna2323/App/blob/main/src/components/TaskHeaderActionButton.tsx#L37). If there is no use case to hide ActionButton please revert the same change on [ReportScreen](https://github.com/Expensify/App/pull/53749/files#diff-68b13bc8344820144538408415f307efee198ccc63977ba318c4234102ec3917R788)